nautobot/dcim/forms.py

```diff
@@ -1755,6 +1761,8 @@ def __init__(self, *args, **kwargs):
                 self.initial["location"] = self.instance.parent_bay.device.location_id
                 self.initial["rack"] = self.instance.parent_bay.device.rack_id
 
+            self.initial["vrfs"] = self.instance.vrfs.values_list("id", flat=True)
```

Comment:
I'm wondering why this is necessary here but when I made a similar change to `InterfaceForm` in #3399 I didn't have to do this.

Reply:
Talking to @jathanism it looks like it's because this is the reverse relationship. The forward relationship on the `VRFform` didn't require these.